server/helpers/jwt.js

```diff
@@ -0,0 +1,23 @@
+import jwt from "jsonwebtoken";
+
+/**
+ * @description This method accepts user object and 
+ returns the requested token for the user
+ * @param {object}  user
+ * @returns {String}  jwtToken  
+ */
+const jwtEncode = user => {
+  const { id, email, lastName, firstName, isAdmin } = user;
+  return jwt.sign(
+    {
+      id,
```

Comment:
Why I this implemented in your task, I only said social Media OAuth

Reply:
The response required we send a token response.
<img width="880" alt="screenshot 2019-02-09 at 12 13 41 pm" src="https://user-images.githubusercontent.com/41480580/52519982-2de75d80-2c64-11e9-91c3-8fd881c32e6c.png">

Reply:
To clarify, you log in with social auth and you should get a json web token